<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KillDialog</class>
 <widget class="QDialog" name="KillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">Not Responding</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layout_message">
     <item>
      <widget class="QLabel" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">ICON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_message">
       <property name="text">
        <string notr="true">MESSAGE</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KCollapsibleGroupBox" name="kcollapsiblegroupbox">
     <property name="title">
      <string>&amp;Details</string>
     </property>
     <property name="expanded">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_pid">
        <property name="text">
         <string notr="true">PID</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_hostname">
        <property name="text">
         <string notr="true">HOSTNAME</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_status" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="KBusyIndicatorWidget" name="busyIndicator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string notr="true">TERMINATING</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::NoButton</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KBusyIndicatorWidget</class>
   <extends>QWidget</extends>
   <header>kbusyindicatorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KCollapsibleGroupBox</class>
   <extends>QWidget</extends>
   <header>kcollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
